README.md

```diff
@@ -175,14 +190,44 @@ This metrics came from `metrics` view table, they are calculated by the followin
 | **MTTS** | Mean Time To Solution | `total solutiontime` / `total incidents` |
 | **MTTR** | Mean Time To Recovery | `total downtime` / `total incidents` |
 
+### Alerts
+
+Alerts are useful for notifying the status of incidents. Notifications can be used in different situations, such as requesting an update of the incident status or finalizing the post-mortem. For this you can use the CLI `notify` on your CronJob service.
+
+To use the CLI you need to build the binary file:
+
+```shell
+go build -o notify cmd/notify/main.go
+```
+
+#### Example of use
+
+```shell
+SHELL=/bin/bash
+BASH_ENV=/app/.env
+
+# At 16:30 on every day-of-week from Monday through Friday send a report to a channel with all incidents not closed
+30 16 * * 1-5 root /app/notify --type=report --to=YOUR_SLACK_CHANNEL_ID --status=all
+
+# At every 30th minute sends a status update request alert for all open incidents
+0/30 * * * * root /app/notify --type=channels --status=open
+
+# At 13:30 on every day-of-week from Monday through Friday sends a post-mortem request alert for all resolved incidents
+30 13 * * 1-5 root /app/notify --type=channels --status=resolved
+```
+
 ## Contributing
+
 Thanks for being interested in contributing! We’re so glad you want to help! Please take a little bit of your time and look at our [contributing guidelines](/docs/CONTRIBUTING.md). All type of contributions are welcome, such as bug fixes, issues or feature requests.
 
 ## Code of Conduct
+
 Everyone interacting in the Hellper project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](/docs/CODE_OF_CONDUCT.md).
 
-## Need help?
+## Need help
```

Comment:
Why the "?" was removed? I believe it makes sense as the documentation is offering a solution whenever the person who's exploring the tool is faced with a question, even if it's a "how to" one.

Reply:
Markdownlint removed this, I did not notice :/
https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md026